### ISSUE TITLE
Add models instead of storing in list

### ISSUE THREAD

**Issue by gabaji** [subodhk01/a2oj#42]
I noticed that you are storing problems in life ladders.py You should use a database for it which will make it easier to add new problems using the admin interface. That way you can also add difficulty levels according to rating of the problem and much more.

**Comment by subodhk01** [comment]
Hey @gabaji that is a great idea. By any chance would you like to work on this issue?

**Comment by gabaji** [comment]
Yes sure I will after my exams. Can you share your discord handle?

**Comment by subodhk01** [comment]
Yeah sure, subodhk#2403

**Comment by EngAhmedElkady** [comment]
I will handle this Feature

**Comment by EngAhmedElkady** [comment]
> Yes sure I will after my exams. Can you share your discord handle?

it is a leader from ahmed Ali, we don't need to add more problem it is template
